YourEasyRent/Controllers/ProductController.cs

```diff
@@ -81,10 +88,10 @@ public async Task<ActionResult<Product>> GetProductByName(string name)
             try
             {
                 var nameProduct = await _repository.GetByName(name);
-                if (nameProduct == null)
-                {
-                    return NotFound();
-                }
+                //if (nameProduct == null)
```

Comment:
Just delete it

Reply:
"if  (nameProduct == null)" and test removed